cms/src/test/java/uk/nhs/digital/apispecs/IsAnyTrueHelperTest.java

```diff
@@ -0,0 +1,150 @@
+import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import uk.nhs.digital.apispecs.swagger.IsAnyTrueHelper;
+
+import java.io.IOException;
+import java.util.*;
+
+public class IsAnyTrueHelperTest {
+
+    private static final String TEMPLATE_CONTENT_FROM_THE_POSITIVE_BLOCK = RandomStringUtils.random(10);
+    private static final String TEMPLATE_CONTENT_FROM_THE_INVERSE_BLOCK = RandomStringUtils.random(10);
+
+    private final IsAnyTrueHelper isAnyTrueHelper = IsAnyTrueHelper.INSTANCE;
+
+    @Mock private Options.Buffer buffer;
+    private String item = "some-item";
+
+    @Before
+    public void setUp() throws Exception {
+        initMocks(this);
+    }
+
+    @Test
+    public void rendersBlockWhenCollectionHasAtLeastOneTrueItem() throws IOException {
+
+        Boolean[] params = {false,false,true,false,false};
+        StubOptions options = new StubOptions(params);
+
+        // when
+        final Options.Buffer actualBuffer = (Options.Buffer) isAnyTrueHelper.apply(item, options);
+
+        // then
+        assertThat("Returns Options.Buffer",
+            buffer,
+            sameInstance(actualBuffer)
+        );
+
+        then(actualBuffer).should().append(TEMPLATE_CONTENT_FROM_THE_POSITIVE_BLOCK);
+    }
+
+    @Test
+    public void rendersBlockWhenCollectionHasNoTrueItem() throws IOException {
+
+        Boolean[] params = {false,false,false,false,false};
+        StubOptions options = new StubOptions(params);
+
+        // when
+        final Options.Buffer actualBuffer = (Options.Buffer) isAnyTrueHelper.apply(item, options);
+
+        // then
+        assertThat("Returns Options.Buffer",
+            buffer,
+            sameInstance(actualBuffer)
+        );
+
+        then(actualBuffer).should().append(TEMPLATE_CONTENT_FROM_THE_INVERSE_BLOCK);
+    }
+
+    @Test
+    public void throwExceptionWhenCollectionIsEmpty() throws IOException {
+
+        Boolean[] params = {};
+        StubOptions options = new StubOptions(params);
+
+        try {
+            // when
+            isAnyTrueHelper.apply(item, options);
+
+        } catch (Exception e) {
+
+            // then
+            assertThat("Exception in execution",
+                e.getMessage(),
+                containsString("At least one parameter is required but none was provided.")
+            );
+        }
+    }
+
+    @Test
+    public void throwExceptionWhenCollectionIsNull() throws IOException {
+
+        Boolean[] params = null;
+        StubOptions options = new StubOptions(params);
+
+        try {
+            // when
+            isAnyTrueHelper.apply(item, options);
+
+        } catch (Exception e) {
+
+            // then
+            assertThat("Exception in execution",
+                e.getMessage(),
+                containsString("At least one parameter is required but none was provided.")
+            );
+        }
+    }
+
```

Comment:
This test case verifies that a validation exception is thrown when the value of `options.params` is `null`.

Which test case verifies that a validation exception is thrown when either of its elements is `null`? In other words, which test case targets `Validation.noNullElements`?

Reply:
following test case check for when either of element in collection is null.

throwExceptionWhenAtLeastOneIsNull()